integration_test/gce/gce_testing.go

```diff
@@ -1031,7 +1031,11 @@ func attemptCreateInstance(ctx context.Context, logger *log.Logger, options VMOp
 	if IsSUSE(vm.Platform) {
 		// Set download.max_silent_tries to 5 (by default, it is commented out in
 		// the config file). This should help with issues like b/211003972.
-		_, err := RunRemotely(ctx, logger, vm, "", "sudo sed -i -E 's/.*download.max_silent_tries.*/download.max_silent_tries = 5/g' /etc/zypp/zypp.conf")
+		backoffPolicy := backoff.WithContext(backoff.WithMaxRetries(backoff.NewConstantBackOff(5*time.Second), 5), ctx) // 5 attempts.
```

Comment:
Nit: let's add constants for the backoff duration and retries, e.g. add this to line 156:

```go
	slesInitMaxAttempts     = 5
	slesInitBackoffDuration = 5 * time.Second
```

LGTM otherwise 👍